Src/WitsmlExplorer.Frontend/components/ContextMenus/TubularSidebarContextMenu.tsx

```diff
@@ -43,15 +45,15 @@ const TubularSidebarContextMenu = (props: TubularSidebarContextMenuProps): React
           <StyledIcon name="refresh" color={colors.interactive.primaryResting} />
           <Typography color={"primary"}>Refresh tubular</Typography>
         </MenuItem>,
-        <MenuItem key={"copy"} onClick={() => onClickCopy(selectedServer, [tubular], dispatchOperation)}>
+        <MenuItem key={"copy"} onClick={() => copyObjectOnWellbore(selectedServer, [tubular], dispatchOperation, ObjectType.Tubular)}>
           <StyledIcon name="copy" color={colors.interactive.primaryResting} />
           <Typography color={"primary"}>Copy tubular</Typography>
         </MenuItem>,
         <MenuItem key={"paste"} onClick={() => onClickPaste(servers, serverUrl, dispatchOperation, orderCopy)} disabled={tubularComponentReferences === null}>
           <StyledIcon name="paste" color={colors.interactive.primaryResting} />
           <Typography color={"primary"}>{menuItemText("paste", "tubular components", tubularComponentReferences?.tubularComponentUids)}</Typography>
```

Comment:
A leftover not related to changes in this PR, but this text gives double ss on plural.
![image](https://user-images.githubusercontent.com/1442452/194537588-8cc9015c-82e7-4b26-8a0a-9a2e3ccada25.png)